### PR TITLE
added my animation box-rotate-animation

### PR DESCRIPTION
**Pull request by Life1sOk**
Hi, thanke you. Have a nice day

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -5868,6 +5868,12 @@ let cards = [
     imageLink: './Art/animationzone/ezgif.com-gif-maker',
     author: 'Erez',
     githubLink: 'https://github.com/ErezAvni9'
+  },
+  {
+    pageLink: './Art/box-rotate-animation/index.html',
```
Comment (donmarvex) [inline include.js:5873]: Please add the "artName" property and give your art a nice name!

**Review by donmarvex — CHANGES_REQUESTED** [review]

**Comment by donmarvex** [comment]
Hi @Life1sOk , thank You for making the changes but the contribution guidelines require you to perform this step outlined below;

**_5. Get a screen recording of your finished work, and make a gif! Try to crop it so that it looks good as a smallish (preferably squarish) image. Save this in your directory, together with your HTML and CSS files. Static screenshots are also acceptable.
If you don't add a gif/screenshot, the website won't show your animation._**

So I'd really like you to make this change in order for your animation to be displayed on the web application.

**Review by donmarvex — APPROVED** [review]
Everything looks great! Thank You.

**Comment by donmarvex** [comment]
All good here @sophiabrandt

**Comment by Life1sOk** [comment]
Thank you for your help

**Pull request merged**